### PR TITLE
Update contributing page

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
## What kind of change does this PR introduce?

Update contributing page.

### CODE REVIEW

File: web/docs/handbook/contributing.mdx [hunk 1]
```diff
@@ -12,10 +12,10 @@ Want to contribute? Why not jump into our GitHub repo and:
 
 - [Sponsor Supabase](https://github.com/sponsors/supabase).
 - [Answer Discussions](https://github.com/supabase/supabase/discussions).
-- Submit an issue.
+- [Submit an issue](https://github.com/supabase/supabase/issue).
```
Comment (awalias) [inline web/docs/handbook/contributing.mdx:15]: this one depends on the specific repo I think. Which brings me to something that would  be useful for this page...

we could introduce a small selection of our repos here, maybe a link and one line explanation of each of supabase/supabase, supabase/supabase-js, and supabase/gotrue followed by "explore all repos" which links to https://github.com/orgs/supabase/repositories

File: web/docs/handbook/contributing.mdx [hunk 2]
```diff
@@ -12,10 +12,10 @@ Want to contribute? Why not jump into our GitHub repo and:
...
 - Report a performance issue or a part of the documentation that you find confusing.
-- Create a pull request.
-- [Translate our Readme](https://github.com/supabase/supabase/issues/1341).
+- [Create a pull request](https://github.com/supabase/supabase/pulls).
```
Comment (awalias) [inline web/docs/handbook/contributing.mdx:17]: this one will also depend on which repo they're contributing to so maybe not required as a link just yet

**Review by awalias — CHANGES_REQUESTED** [review]

**Comment by Isaiah-Hamilton** [comment]
@awalias is this what you wanted. Thoughts?
Screen Shot 2022-01-12 at 8 50 28 PM

**Review by kiwicopple — APPROVED** [review]

File: about/docs/handbook/contributing.mdx [hunk 3]
```diff
@@ -10,9 +10,17 @@ Want to contribute? Why not jump into our GitHub repo and:
 
 - [Sponsor Supabase](https://github.com/sponsors/supabase).
 - [Answer Discussions](https://github.com/supabase/supabase/discussions).
-- [Submit an issue](https://github.com/supabase/supabase/issue)
+- Submit an issue.
+    - [supabase/supabase](https://github.com/supabase/supabase/issues)
```
Comment (awalias) [inline about/docs/handbook/contributing.mdx:14]: thanks for the update @Isaiah-Hamilton !

I was actually thinking a separate section above the current list, something like:

```
<h1>How to contribute</h1>

Supabase has many [open source repos](https://github.com/orgs/supabase/repositories). Some of the more popular ones are:
  - [Supabase](link) - Our Dashboard, Websites, and example apps
  - [Supabase-js](link) - A clientside library written in Typescript
  - [Gotrue](link) - Our auth server, written in Go
  - [Realtime](link) - A server for streaming changes from Postgres, written in Elixir 
  - [browse all]((https://github.com/orgs/supabase/repositories))

Here are some ways you can contribute:
  - Sponsor Supabase
  - Start a discussion
  - Submit an issue to one of our [repos]((https://github.com/orgs/supabase/repositories))
... 
```

**Review by awalias — CHANGES_REQUESTED** [review]

**Comment by Isaiah-Hamilton** [comment]
@awalias All updated.
Screen Shot 2022-01-14 at 8 53 23 PM

**Review by awalias — APPROVED** [review]
love it!

**Pull request merged**